cmake/features.cmake

```diff
@@ -154,6 +154,8 @@ ie_option(ENABLE_OV_IR_FRONTEND "Enable IR FrontEnd" ON)
 ie_dependent_option(ENABLE_OV_TF_FRONTEND "Enable TensorFlow FrontEnd" ON "protoc_available" OFF)
 ie_dependent_option(ENABLE_SYSTEM_PROTOBUF "Use system protobuf" OFF
     "ENABLE_OV_ONNX_FRONTEND OR ENABLE_OV_PADDLE_FRONTEND OR ENABLE_OV_TF_FRONTEND;BUILD_SHARED_LIBS" OFF)
+ie_dependent_option(ENABLE_SHUTDOWN_PROTOBUF "Shutdown protobuf when unload the frontend shared libraries" ON
```

Comment:
Is there a way to make it as 'internal' variable for this purpose? I'm not sure that we should give an option to user to 'cmake -DENABLE_SHUTDOWN_PROTOBUF=OFF'
This variable shall be 'ON' if some of protobuf frontends are enabled, and be always OFF otherwise.

Reply:
why do we even need this option? Why it cannot be detected automatically?

Reply:
User doesn't need to set '-DENABLE_SHUTDOWN_PROTOBUF=OFF'. By default it is ON, if onnx/paddle/tf frontends are all not enabled or it is not build as shared library, it will be OFF.

Reply:
@ilya-lavrenov , It can be detected automatically. It provides user an option to OFF this feature. If you think we didn't need it. I can change it to an internal signal and detected automatically.

Reply:
@ilya-lavrenov , @nosovmik , Here is the PR to remove this ie option in case you think it is not needed. https://github.com/openvinotoolkit/openvino/pull/9619